src/service.js

```diff
@@ -363,6 +255,63 @@ export function getServicePromiseOrNullForDoc(nodeOrDoc, id) {
       id);
 }
 
+/**
+ * @param {!Window} embedWin
+ * @param {string} id
+ * @return {?Object}
+ */
+function getExistingServiceForEmbedWinOrNull(embedWin, id) {
```

Comment:
Can you undo the change in position for these methods? It makes the diff harder to read. 

In general, recommend minimizing diffs in sensitive changes like this PR.